docker-compose.yml

```diff
@@ -134,6 +135,8 @@ services:
             - JIBRI_BREWERY_MUC
             - JIBRI_PENDING_TIMEOUT
             - TZ
+            - ENABLE_OCTO
+            - OCTO_BS_STRATEGY
```

Comment:
change it to `OCTO_BRIDGE_SELECTION_STRATEGY`.

Reply:
Done